disnake/ext/commands/interaction_bot_base.py

```diff
@@ -68,14 +72,28 @@
 _log = logging.getLogger(__name__)
 
 
+class _Diff(TypedDict):
+    no_changes: List[ApplicationCommand]
+    upsert: List[ApplicationCommand]
+    edit: List[ApplicationCommand]
+    delete: List[ApplicationCommand]
+    delete_ignored: NotRequired[List[ApplicationCommand]]
+
+
+class AppCommandMetadata(NamedTuple):
```

Comment:
This class is unused in this PR

Reply:
https://github.com/DisnakeDev/disnake/pull/806/commits/b6df4910977e7cca86b7a4f703202c94cdb4c40e